pkg/build/api/validation/validation.go

```diff
@@ -461,3 +461,26 @@ func isValidURL(uri string) bool {
 	_, err := url.Parse(uri)
 	return err == nil
 }
+
+func ValidateBuildLogOptions(opts *buildapi.BuildLogOptions) fielderrors.ValidationErrorList {
+	allErrs := fielderrors.ValidationErrorList{}
+
+	// TODO: Replace by validating PodLogOptions once it's included in BuildLogOptions
```

Comment:
> build the PodLogOptions and call teh correct validate method.

The logic to build it should probably be the `api/helpers.go` since somewhere else will need to do it too